x-pack/plugins/cloud_security_posture/public/components/no_findings_states.tsx

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiLoadingLogo, EuiButton, EuiEmptyPrompt } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useCspBenchmarkIntegrations } from '../pages/benchmarks/use_csp_benchmark_integrations';
+import { useCISIntegrationPoliciesLink } from '../common/navigation/use_navigate_to_cis_integration_policies';
+import { NO_FINDINGS_STATUS_TEST_SUBJ } from './test_subjects';
+import { CloudPosturePage } from './cloud_posture_page';
+import { useCspSetupStatusApi } from '../common/api/use_setup_status_api';
+
+const REFETCH_INTERVAL = 20000;
```

Comment:
nit: name this `REFETCH_INTERVAL_MS` so it will be clear what the unit of measurement is here